packages/amplify_datastore/example/lib/models/Blog.dart

```diff
@@ -64,30 +76,32 @@ class Blog extends Model {
     var buffer = new StringBuffer();
 
     buffer.write("Blog {");
-    buffer.write("id=" + id + ", ");
-    buffer.write("name=" + name);
+    buffer.write("id=" + "$id" + ", ");
```

Comment:
nit: `buffer.write("id=$id, ");` ditto to other string interpolation

Reply:
thanks this is a codegen change.  If the codegen team has time I can ask them to make this change.